src/core/lombok/bytecode/PoolConstantsApp.java

```diff
@@ -66,7 +66,7 @@
 		CmdReader<CmdArgs> reader = CmdReader.of(CmdArgs.class);
 		CmdArgs args;
 		try {
-			args = reader.make(raw.toArray(new String[0]));
+			args = reader.make(raw.toArray(new String[raw.size()]));
```

Comment:
And take care, the best result is ToArrayBench.zero_sized_array with 40 000 executions by ms vs ToArrayBench.sized_array_from_list who is "only" 32 000 executions by ms.